tests/README.md

```diff
@@ -17,7 +17,17 @@ Tests do exist, but their coverage generally needs improvement...
 
 ## Running a test
 
-To run a test, simply run `python <name-of-file>.py`
+To run all unit tests, run:
+
+```shell
+bash unittests/run_all.bash
```

Comment:
I would heavily encourage `pytest unittests/` over this - we're already using this in CI and it has been working well.

Reply:
Thanks I didn't know that was possible when not all tests are using `pytest`